kotlin/http4k/build.gradle

```diff
@@ -0,0 +1,38 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlin_version}"
+        classpath "com.github.jengelman.gradle.plugins:shadow:${shadow_version}"
+    }
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+apply plugin: 'kotlin'
+apply plugin: "com.github.johnrengelman.shadow"
+apply plugin: 'application'
+
+group 'benchmark'
+version '0.0.1'
+mainClassName = "org.http4k.benchmark.BenchmarkServerKt"
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
```

Comment:
is there a possible issue `here` ? I mean all `kotlin` **frameworks** runs on `jvm` **11**, but kotlin stlib targets `jvm` **8** @daviddenton

Reply:
this is fine. it's just the bytecode level that is targetted.

Reply:
it is 👍 , since it's working
however, is there not any feature in `jvm` that you can't take advantage of ?

Reply:
By running on JVM 11 you get to take advantage of all the sweet new JVM features like garbage collection improvements etc. Compiling against JDK 11 will give you access to the new JDK libraries, but we don't care about those because we use the Kotlin StdLib instead (which has overall much better APIs and type system).

Anyway - glad it's working!